### ISSUE TITLE
Rulings incorrect for Mana Reflection (perhaps due to mana symbols?)

### ISSUE THREAD

**Issue by Fryyyyy** [mtgjson/mtgjson#348]
```
"text": "Mana Reflection doesn't cause permanents you tap for mana to produce one extra mana -- rather, it doubles the type and amount of mana that permanent would produce. For example, if you tap Mystic Gate for {U} instead."
```
should read
```
Mana Reflection doesn’t cause permanents you tap for mana to produce one extra mana — rather, it doubles the type and amount of mana that permanent would produce. For example, if you tap Mystic Gate for {W}{U}, it will produce {W}{W}{U}{U} instead.
```

and
```
"text": "If you have Mana Reflection on the battlefield and tap a permanent for mana, but another replacement effect would change the type and/or amount of mana i
t would produce (such as Contamination, which says \"If a land is tapped for mana, it produces {B}."
```
should read
```
If you have Mana Reflection on the battlefield and tap a permanent for mana, but another replacement effect would change the type and/or amount of mana it would produce (such as Contamination, which says “If a land is tapped for mana, it produces {B} instead of any other type and amount”), you decide what order to apply those effects in. In this example, applying Mana Reflection first and Contamination second would result in  {B}; applying Contamination first and Mana Reflection second would result in {B}{B}.
```

**Comment by Psithief** [comment]
This looks like a greedy regular expression's work.

**Comment by Psithief** [comment]
This was actually fixed in #328. It hasn't been published yet.
https://raw.githubusercontent.com/mtgjson/mtgjson/a7c605cf4a3540aba45040b174d84fd0809be1f0/json/SHM.json

**Comment by gwax** [comment]
This has been fixed and released.
